### DISCUSSION TITLE
iPad pro m1 or m2

### DISCUSSION THREAD

**Discussion by Airmancooma** [utmapp/UTM#5196] [UTM Help (iOS)]
Hi guys I have a question!

A good opportunity has come up and I could buy an ipad pro m1 1tb (16gb ram) or an m2 256gb (8gb ram). My question is will windows 10 run on 4gb ram? Or not? How much difference in performance? (I am thinking about virtualization)

**Comment by alistera17** [comment]
I put huge amounts of research in to this, running UTM/QEMU on an iPad Pro with an M1 as needed to run a fund/wealth management platform which was windows based, Remote Desktop took too much bandwidth and too slow for chart zoom so wasn't really an option, apparently you can do this with an iPhone which you have to tether to the iPad, have all the equipment but in the end we couldn't be bothered and they migrated the platform to web based, realtime web charting these days is much better than a few years ago, not sure having a $2k iPad tethered to a $2k iPhone in a coffee shop is the way to go!

**Comment by kkebo** [comment]
I had used Windows 11 on UTM with JIT enabled on my M2 iPad Pro (16GB RAM). But installation was much slower than when using the Virtualization framework on macOS, and crashes and disk image corruption occurred frequently during use.

If you want to use the Virtualization framework on iPadOS, you have to install UTM via TrollStore. TrollStore only supports iPadOS 14.0 - 15.1.1, so you can't use it on the M2 iPad Pro because its minimum OS version is 16.1. And also, the M1 iPad Pro's OS version also may be newer than that when you buy it. In such case, you have to jailbreak your device to downgrade the OS version.

Thus, I don't recommend you to use an iPad Pro to run a Windows VM at least now.

**Comment by alistera17** [comment]
Yes, it was very specific what iPadOS versions you can use and Apple do not list the exact version on their packaging, you can only do it without rooting the iPad (not ideal) by using a tether via Jitterbug, this is the only combination we looked, it's all very complicated and easier if you can find a way to refactor the app.

Regards,
Alister
________________________________
